argmin/src/solver/quasinewton/lbfgs.rs

```diff
@@ -61,6 +68,8 @@ pub struct LBFGS<L, P, G, F> {
     tol_grad: F,
     /// Tolerance for the stopping criterion based on the change of the cost stopping criterion
     tol_cost: F,
+    /// Coefficient of L1-regularization
+    l1_coeff: F,
```

Comment:
I think I would prefer this to be `Option<F>`, where `None` represents no L1 regularization.